upgrade/1.2/Stage_0_Prerequisites.md

```diff
@@ -119,68 +128,78 @@ CSM 1.2 introduces the bifurcated CAN (BICAN) as well as network configuration c
 upgrade and its sequence of events, see the [SLS upgrade `README`](scripts/sls/README.SLS_Upgrade.md).
 
 The SLS data upgrade is a critical step in moving to CSM 1.2. Upgraded SLS data is used in DNS and management network configuration. For details to aid in understanding and
-decision making, see the [Management Network User Guide](../../operations/network/management_network/README.md).
+decision making, see the [Management Network User Guide](../../operations/network/management_network/index.md).
 
 One detail which must not be overlooked is that the existing Customer Access Network (CAN) will be migrated or retrofitted into the new Customer Management Network (CMN) while
 minimizing changes. A new CAN (or CHN) network is then created. Pivoting the existing CAN to the new CMN allows administrative traffic (already on the CAN) to remain as-is while
-moving standard user traffic to a new site-routable network.
+moving standard user traffic to a new site-routable network. You can read more about this, as well as steps to ensure undisrupted access to UANs during upgrade, in
+[Plan and coordinate network upgrade](plan_and_coordinate_network_upgrade.md).
 
 > **Important:** If this is the first time performing the SLS update to CSM 1.2, review the [SLS upgrade `README`](scripts/sls/README.SLS_Upgrade.md) in order to ensure
 the correct options for the specific environment are used. Two examples are given below. To see all options from the update script, run `./sls_updater_csm_1.2.py --help`.
 
 ### Retrieve SLS data as JSON
 
-1.(`ncn-m001#`) Obtain a token.
+1. Obtain a token.
 
    ```bash
-   export TOKEN=$(curl -s -k -S -d grant_type=client_credentials -d client_id=admin-client \
+   ncn-m001# export TOKEN=$(curl -s -k -S -d grant_type=client_credentials -d client_id=admin-client \
                                 -d client_secret=`kubectl get secrets admin-client-auth -o jsonpath='{.data.client-secret}' | base64 -d` \
                                 https://api-gw-service-nmn.local/keycloak/realms/shasta/protocol/openid-connect/token | jq -r '.access_token')
    ```
 
-1.(`ncn-m001#`) Create a working directory.
+1. Create a working directory.
 
    ```bash
-   mkdir /root/sls_upgrade && cd /root/sls_upgrade
+   ncn-m001# mkdir /root/sls_upgrade && cd /root/sls_upgrade
    ```
 
-1.(`ncn-m001#`) Extract SLS data to a file.
+1. Extract SLS data to a file.
 
    ```bash
-   curl -k -H "Authorization: Bearer ${TOKEN}" https://api-gw-service-nmn.local/apis/sls/v1/dumpstate | jq -S . > sls_input_file.json
+   ncn-m001# curl -k -H "Authorization: Bearer ${TOKEN}" https://api-gw-service-nmn.local/apis/sls/v1/dumpstate | jq -S . > sls_input_file.json
    ```
 
 ### Migrate SLS data JSON to CSM 1.2
 
+<<<<<<< HEAD
```

Comment:
This conflict marker should be removed. Looks like lines 165-167 can be deleted.